Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:5.7.1
+
+# Install libsass
+RUN apt-get update
+RUN apt-get install -y automake libtool ca-certificates build-essential
+RUN wget https://github.com/sass/libsass/archive/3.3.3.tar.gz && \
```

Comment:
so we can just delete that line, and then feed the docker script into AWS?

https://aws.amazon.com/getting-started/tutorials/deploy-docker-containers/